types/did_doc_metadata_test.go

```diff
@@ -26,32 +26,31 @@ func TestNewResolutionDidDocMetadata(t *testing.T) {
 	}
 
 	validMetadataResource := DereferencedResource{
-		ResourceURI:       validDid + RESOURCE_PATH + resourceHeader.Id,
-		CollectionId:      resourceHeader.CollectionId,
-		ResourceId:        resourceHeader.Id,
-		Name:              resourceHeader.Name,
-		ResourceType:      resourceHeader.ResourceType,
-		MediaType:         resourceHeader.MediaType,
-		Created:           resourceHeader.Created,
-		Checksum:          FixResourceChecksum(resourceHeader.Checksum),
+		ResourceURI:       validDid + RESOURCE_PATH + resourceMetadata.Id,
+		CollectionId:      resourceMetadata.CollectionId,
+		ResourceId:        resourceMetadata.Id,
+		Name:              resourceMetadata.Name,
+		ResourceType:      resourceMetadata.ResourceType,
+		MediaType:         resourceMetadata.MediaType,
+		Created:           resourceMetadata.Created,
+		Checksum:          FixResourceChecksum(resourceMetadata.Checksum),
```

Comment:
No need for the fix anymore.

Reply:
The checkSum is in bytes, the Fix just converts it to a string for the resolved output.